mcs/build/profiles/mobile_static.make

```diff
@@ -56,3 +56,11 @@ endif
 
 ALWAYS_AOT = yes
 
+INVARIANT_AOT_OPTIONS=nimt-trampolines=900,ntrampolines=8000,nrgctx-fetch-trampolines=256
+
+ifndef MONO_DISABLE_GSHAREDVT
+INVARIANT_AOT_OPTIONS:=$(INVARIANT_AOT_OPTIONS),ngsharedvt-trampolines=2800
```

Comment:
Could you add some comments on how you picked those numbers and where do they show up?